projects/core/src/cart/cart-persistence.module.ts

```diff
@@ -2,13 +2,14 @@ import { APP_INITIALIZER, ModuleWithProviders, NgModule } from '@angular/core';
 import { ActionReducer, MetaReducer, META_REDUCERS } from '@ngrx/store';
 import { tap } from 'rxjs/operators';
 import { ConfigInitializerService } from '../config/config-initializer/config-initializer.service';
+import { Config } from '../config/config-tokens';
 import { MultiCartStatePersistenceService } from './services/multi-cart-state-persistence.service';
 import { activeCartInitialState } from './store/reducers/multi-cart.reducer';
 
 export function cartStatePersistenceFactory(
   cartStatePersistenceService: MultiCartStatePersistenceService,
   configInit: ConfigInitializerService
-) {
+): () => Promise<Config> {
```

Comment:
Headsup: In `develop` this file is placed in the new cart lib. But in 4.3.x it still lives in the core lib.
See the [corresponding file](https://github.com/SAP/spartacus/pull/15923/files#diff-b7392b6530ed0e2819d57bac03344b40f9e03b3f2d7a6c5c5c7e3afa739b3819) in the original PR to develop.

No action needed.